### PR TITLE
#VP-2845 exp profile

### PR DESCRIPTION
**Pull request by kostyrin**

### CODE REVIEW

File: VirtoCommerce.Storefront.Model/Customer/Contracts/CreateOrganizationResponseDto.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+namespace VirtoCommerce.Storefront.Model.Customer.Contracts
+{
+    public class CreateOrganizationResponseDto
```
Comment (eMazeika) [inline VirtoCommerce.Storefront.Model/Customer/Contracts/CreateOrganizationResponseDto.cs:3]: Do we actually need classes for each method response?

File: VirtoCommerce.Storefront.Model/Customer/Contracts/UpdateOrganizationResponseDto.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+namespace VirtoCommerce.Storefront.Model.Customer.Contracts
+{
+    public class UpdateOrganizationResponseDto
```
Comment (eMazeika) [inline VirtoCommerce.Storefront.Model/Customer/Contracts/UpdateOrganizationResponseDto.cs:3]: Do we actually need classes for each method response?

File: VirtoCommerce.Storefront/Domain/Customer/ExperienceMemberService.cs [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+                        contact.Name,
+                        contact.FullName,
+                        contact.FirstName,
+                        contact.LastName,
+                        contact.MiddleName,
+                        contact.Salutation,
+                        contact.PhotoUrl,
+                        contact.TimeZone,
+                        contact.DefaultLanguage,
+                        contact.Addresses
```
Comment (eMazeika) [inline VirtoCommerce.Storefront/Domain/Customer/ExperienceMemberService.cs:56]: missing conversion toDto prevents to pass addresses, isn't it?

File: VirtoCommerce.Storefront/Domain/Customer/MemberConverter.cs [hunk 4]
```diff
@@ -103,6 +104,19 @@ public static Organization ToOrganization(this OrganizationRegistration orgRegis
             return organization;
         }
 
+        public static Organization ToOrganization(this OrganizationDto organizationDto)
+        {
+            var organization = new Organization
+            {
+                Name = organizationDto.Name,
```
Comment (eMazeika) [inline VirtoCommerce.Storefront/Domain/Customer/MemberConverter.cs:111]: is only 1 property enough?

File: VirtoCommerce.Storefront.Model/Customer/Contracts/CreateOrganizationResponseDto.cs (hunk 1 above)
Comment (kostyrin) [reply VirtoCommerce.Storefront.Model/Customer/Contracts/CreateOrganizationResponseDto.cs:3]: I suppose , yes. look at ExperienceCartBuilder

File: VirtoCommerce.Storefront.Model/Customer/Contracts/UpdateOrganizationResponseDto.cs (hunk 2 above)
Comment (kostyrin) [reply VirtoCommerce.Storefront.Model/Customer/Contracts/UpdateOrganizationResponseDto.cs:3]: I suppose , yes. look at ExperienceCartBuilder

File: VirtoCommerce.Storefront.Model/Customer/Contracts/MemberDto.cs [hunk 5]
```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+
+namespace VirtoCommerce.Storefront.Model.Customer.Contracts
```
Comment (tatarincev) [inline VirtoCommerce.Storefront.Model/Customer/Contracts/MemberDto.cs:3]: You don't need such separated Dto types, you can deserialize directly to Storefront.Customer domain types

File: VirtoCommerce.Storefront/Domain/Customer/MemberConverter.cs (hunk 4 above)
Comment (kostyrin) [reply VirtoCommerce.Storefront/Domain/Customer/MemberConverter.cs:111]: look at line 94

File: VirtoCommerce.Storefront.Model/Customer/Contracts/MemberDto.cs (hunk 5 above)
Comment (eMazeika) [reply VirtoCommerce.Storefront.Model/Customer/Contracts/MemberDto.cs:3]: The Dtos are needed, as the models are not identical and migth change. There is issue returning Customer.Organization at the moment

**Pull request merged**